src/portal/src/i18n/lang/ko-kr-lang.json

```diff
@@ -0,0 +1,1943 @@
+        "ENABLE_SUMMARY": "로봇(Robot){{param}}을 활성화하시겠습니까?",
+        "DISABLE_TITLE": "로봇(Robot) 비활성화",
+        "DISABLE_SUMMARY": "로봇(Robot){{param}}을 비활성화 하시겠습니까?",
+        "ENABLE_ROBOT_SUCCESSFULLY": "로봇(Robot)을 성공적으로 활성화했습니다",
+        "DISABLE_ROBOT_SUCCESSFULLY": "로봇(Robot)을 성공적으로 비활성화했습니다",
+        "ROBOT_ACCOUNT": "로봇(Robot) 계정",
+        "PROJECTS": "프로젝트",
+        "ALL_PROJECTS": "모든 프로젝트",
+        "PERMISSIONS": "권한",
+        "REFRESH_SECRET_SUMMARY": "로봇 계정 비밀을 자동으로 새로 고치거나 선택적으로 세부 정보를 열어 새 비밀을 수동으로 지정하세요.",
+        "TOKEN": "시크릿(Secret)",
+        "NEW_TOKEN": "새 시크릿(Secret)",
+        "REFRESH": "새로고침",
+        "PROJECTS_MODAL_TITLE": "로봇(Robot) 계정을 위한 프로젝트",
+        "PROJECTS_MODAL_SUMMARY": "이 로봇(Robot) 계정이 다루는 프로젝트가 있습니다.",
+        "CREATE_ROBOT": "시스템 로봇(Robot) 계정 생성",
+        "CREATE_ROBOT_SUMMARY": "시스템 및 특정 프로젝트에 대한 권한을 포함하는 시스템 로봇(Robot) 계정을 생성합니다",
+        "EDIT_ROBOT": "시스템 로봇(Robot) 계정 편집",
+        "EDIT_ROBOT_SUMMARY": "시스템 및 특정 프로젝트에 대한 권한을 포함하는 시스템 로봇(Robot) 계정을 편집합니다",
+        "EXPIRATION_TIME": "만료 시간",
+        "EXPIRATION_TIME_EXPLAIN": "로봇(Robot) 계정의 토큰 만료 시간(일 단위, 시작 시점은 생성 시간)입니다. 만료되지 않으려면 \"-1\"을 입력하세요.",
+        "EXPIRATION_DEFAULT": "일(기본값)",
+        "EXPIRATION_DAYS": "# 일 지정",
+        "EXPIRATION_NEVER": "만료 안됨",
+        "EXPIRATION_REQUIRED": "유효한 만료 시간은 필수 항목입니다",
+        "COVER_ALL": "모든 프로젝트를 커버",
+        "COVER_ALL_EXPLAIN": "기존 및 향후 모든 프로젝트에 적용하려면 선택하세요.",
+        "COVER_ALL_SUMMARY": "현재 및 미래의 모든 프로젝트가 선택되었습니다.",
+        "RESET_PERMISSION": "모든 프로젝트 권한 재설정",
+        "PERMISSION_COLUMN": "권한",
+        "EXPIRES_AT": "만료 시간",
+        "VIEW_SECRET": "시크릿(SECRET) 새로고침",
+        "LEGACY": "Legacy",
+        "CREATE_PROJECT_ROBOT": "로봇(Robot) 계정 생성",
+        "CREATE_PROJECT_ROBOT_SUMMARY": "이 프로젝트를 위한 로봇(Robot) 계정 생성",
+        "EDIT_PROJECT_ROBOT": "로봇(Robot) 계정 편집",
+        "EDIT_PROJECT_ROBOT_SUMMARY": "이 프로젝트를 위한 로봇(Robot) 계정 편집",
+        "NOT_FOUND": "로봇(Robot)을 찾을 수 없습니다!",
+        "SELECT_ALL": "모두 선택",
+        "UNSELECT_ALL": "모두 선택하지 않음",
+        "ROBOT_ACCOUNT_NAV": "로봇(Robot) 계정",
+        "COVERED_PROJECTS": "프로젝트",
+        "CONFIRM_SECRET": "시크릿(Secret) 확인",
+        "SECRET_AGAIN": "시크릿(Secret)을 다시 입력하세요",
+        "INCONSISTENT": "두 시크릿(Secrets)이 일치하지 않습니다",
+        "NAME_TOOLTIP": "로봇(Robot) 이름은 1~255자(영문 소문자, 숫자, ._- 포함)여야 하며 문자나 숫자로 시작해야 합니다.",
+        "ENABLE_NEW_SECRET": "새(Secret)을 수동으로 지정하려면 이 옵션을 활성화하세요",
+        "DELETE": "삭제",
+        "ARTIFACT_LABEL": "아티팩트(Artifact) 라벨",
+        "SCAN": "스캔",
+        "SCANNER_PULL": "스캐너 풀(PUll)",
+        "SEARCH_BY_NAME": "이름으로 검색(접두사 없이)",
+        "FINAL_PROJECT_NAME_TIP": "최종 프로젝트 로봇 이름은 접두사, 프로젝트 이름, 더하기 기호 및 현재 입력 값으로 구성됩니다.",
+        "FINAL_SYSTEM_NAME_TIP": "최종 시스템 로봇 이름은 접두어와 현재 입력 값으로 구성됩니다.",
+        "PUSH_AND_PULL": "푸쉬(Push)",
+        "PUSH_PERMISSION_TOOLTIP": "푸시 권한은 단독으로 작동할 수 없으며, 풀 권한과 함께 작동해야 합니다.",
+        "STOP": "중지",
+        "LIST": "리스트",
+        "REPOSITORY": "레포티토리",
```

Comment:
I think other technical words are written with tuning forks.
In that case, I think it should be written as ‘레포지토리(Repository)’ rather than ‘저장소(Repository)’.